spec/features/idv/doc_auth/document_capture_spec.rb

```diff
@@ -421,7 +421,7 @@
               submit_images
               message = strip_tags(t('errors.doc_auth.selfie_fail_heading'))
               expect(page).to have_content(message)
-              detail_message = strip_tags(t('doc_auth.errors.alerts.selfie_poor_quality'))
+              detail_message = strip_tags(t('doc_auth.errors.alerts.selfie_not_live'))
```

Comment:
Before updating translations, the text for `'doc_auth.errors.alerts.selfie_poor_quality'` and `'doc_auth.errors.alerts.selfie_not_live'` were the same.  Since they are no longer the same, I had to update this spec.